tests/fixtures/nodes/src/nodes.rs

```diff
@@ -468,6 +469,8 @@ pub fn nodes(_tracing: &Tracing) -> Nodes {
         cleanup::register_child_process(child_process);
     }
 
+    let payments_seed = env::var("TEST_PAYMENTS_SEED").unwrap_or_else(|_| "payments-seed".to_string());
```

Comment:
This used to be `payment-seed`, now it's `payments-seed`. Not a big issue but you'll lose any previously funded accounts in testnet.

Reply:
good catch. thanks!